xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -77,13 +76,13 @@
       SynchronizationContext syncContext,
       ScheduledExecutorService scheduledExecutorService,
       BackoffPolicy.Provider backoffPolicyProvider,
-      Supplier<Stopwatch> stopwatchSupplier) {
+      Stopwatch stopwatch) {
     this.loadStatsManager = checkNotNull(loadStatsManager, "loadStatsManager");
     this.xdsChannel = checkNotNull(xdsChannel, "xdsChannel");
     this.syncContext = checkNotNull(syncContext, "syncContext");
     this.timerService = checkNotNull(scheduledExecutorService, "timeService");
     this.backoffPolicyProvider = checkNotNull(backoffPolicyProvider, "backoffPolicyProvider");
-    this.retryStopwatch = checkNotNull(stopwatchSupplier, "stopwatchSupplier").get();
+    this.stopwatch = checkNotNull(stopwatch, "stopwatch");
```

Comment:
The purpose of using a supplier is to let this class own and solely manage the state of the stopwatch. Passing in a stopwatch instance in the constructor does not prevent the stopwatch instance from being shared and `reset()` outside this class from API perspective. You either call `supplier.get()` inside or outside the constructor, but I think the original version of constructor can not be misused.

Reply:
Yeah, you are right. I can change this back.

Reply:
Changed back.